package.json

```diff
@@ -50,7 +50,7 @@
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-import": "2.24.2",
     "eslint-plugin-prettier": "4.0.0",
-    "knex": "0.21.1",
+    "knex": "^0.95.6",
```

Comment:
We kept this at 0.21 earlier as Objection.js hadn't updated yet, but they have also switched to a newer version now so this should be fine.